UI/Pages/TuitionPartner.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@page "{id}"
 @model TuitionPartner
 @{
-    ViewData["Title"] = Model.Data.Name;
+    ViewData["Title"] = Model.Data!.Name;
```

Comment:
I think it would be safer to use null-coalescing here as we cannot guarantee that `Data` will never be null in future changes.